PrimerasPracticasJS/PalabraAlreves.js

```diff
@@ -0,0 +1,10 @@
+/*PALABRA AL REVES O INVERTIDA*/
+
+let word = 'Hola Mundo'
+function wordInverted(word) {
+
+    let inverted = word.split('').reverse().join('')
+
+    return inverted
```

Comment:
si es un proceso que se hace en una sola linea de codigo, no vale la pena guardar el resultado en una variable, basta con regresar el resultado directo.

```suggestion
    return word.split('').reverse().join('')
```